### ISSUE TITLE
Creating TagGroup not working

### ISSUE THREAD

**Issue by alinr** [rtconner/laravel-tagging#186]
Hi,

when I try this: php artisan tagging:create-group MyTagGroup

I get this error:

```
ErrorException 

  Undefined variable: \Conner\Tagging\Model\TagGroup

  at vendor/rtconner/laravel-tagging/src/Console/Commands/GenerateTagGroup.php:21
    17|     public function handle()
    18|     {
    19|         $groupName = $this->argument('group');
    20| 
  > 21|         $tagGroup = new ${TaggingUtility::tagGroupModelString()};
    22|         $tagGroup->name = $groupName;
    23|         $tagGroup->slug = TaggingUtility::normalize($groupName);
    24| 
    25|         $tagGroup->save();

      +15 vendor frames 
  16  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

Any idea?

**Comment by rtconner** [comment]
Thank you for the report, this was a bug. It is resolved in version 4.0.2 now

**Comment by alinr** [comment]
Thank you, Rob!
